webapp/locales/en.json

```diff
@@ -413,8 +413,8 @@
       "name": "Take action"
     },
     "menu": {
-      "edit": "Edit Post",
-      "delete": "Delete Post",
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-22T09:17:47Z" title="Wednesday, January 22nd 2020, 10:17:47 am +01:00">Jan 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -415,7 +415,7 @@
     "menu": {
       "edit": "Edit Post",
       "delete": "Delete Post",
-      "pin": "Pin post",
+      "pin": "Pin Post",
```


```suggestion
      "pin": "Pin post",
```
we agreed to capitalize the first word in a sentence and lowercase all others....